scripts/system/create/entitySelectionTool/entitySelectionTool.js

```diff
@@ -485,6 +528,7 @@ SelectionManager = (function() {
             } else {
                 delete properties.position;
             }
+            properties.name = getDuplicateAppendedName(properties.name);
```

Comment:
what's the difference between pasting and duplicating?  do we want only duplicating to do this new logic?

Reply:
Duplicate is Move + CTRL... generating a clone of the entity. (this one is right I think)

But just realize that it was not a good implementation for "Paste" (when you do Copy > Paste) because you might also have done "Cut > Paste" and this is going to be wrong in that second case. 
We don't want a cut-paste to increment...  (I would need to check that.)

I envision that more for the "Duplicate" since it's what I do 100% of the time when I use this.
It's probably not too difficult to address Copy versus Cut.

Reply:
I will check for that

Reply:
could you also copy from another domain and paste in a new one?

Reply:
"Duplicate" makes sense to get the suffix (and have it incremented)

"Paste" doesn't. It's not possible for the script to be sure of the origin of what is in the clipboard.
It would not make sense to add the suffix if you paste something copied on another domain.
It would not  make sense to add the suffix to something we cut and then paste (to let say move it somewhere else  in the same or in another domain)
we can paste the cut object N time. If we can track this for the current run, it's impossible the second we close and reopen the creat app.

I will adjust the fix to have this only for "Duplicate".
No suffix for pasted entities.